### ISSUE TITLE
Question about Evaluating FID Metrics

### ISSUE THREAD

**Issue by hxy-123-coder** [Sonettoo/AeroGen#10]
Thanks for your great work！When calculating the FID metric, did you use the standard Inception-v3 model pre-trained on ImageNet for feature extraction, or an Inception-v3 model fine-tuned on remote sensing images?

**Comment by Sonettoo** [comment]
Hello, we use Inception-v3 model fine-tuned on remote sensing images, followed [this work](https://github.com/YonghaoXu/Txt2Img-MHN)
